src/tests/ftest/pool/destroy_tests.py

```diff
@@ -1,746 +1,633 @@
+        self.setup_start_servers_and_clients = False
+        super(DestroyTests, self).setUp()
 
     def test_simple_delete(self):
-        """
-        Test destroying a pool created on a single server, nobody is using
-        the pool, force is not needed.
+        """Test destroying a pool created on a single server.
 
         :avocado: tags=pool,pooldestroy,quick
         """
-        self.hostlist_servers = self.params.get("test_machines1", '/run/hosts/')
-        hostfile_servers = write_host_file.write_host_file(
-            self.hostlist_servers, self.tmp)
-
-        self.agent_sessions = agent_utils.run_agent(self.basepath,
-                                                    self.hostlist_servers)
-        server_utils.run_server(hostfile_servers, self.server_group,
-                                self.basepath)
-
-        setid = self.params.get("setname",
-                                '/run/setnames/validsetname/')
-
+        hostlist_servers = self.hostlist_servers[:1]
+        group_hosts = {self.server_group: hostlist_servers}
+
+        self.start_servers_and_clients(group_hosts)
+        setid = self.params.get("setname", '/run/setnames/validsetname/')
+        # use the uid/gid of the user running the test, these should
+        # be perfectly valid
+        uid = os.geteuid()
+        gid = os.getegid()
+
+        # Create pool with doasctl
+        create_cmd = ('{0} create-pool -m {1} -u {2} -g {3} -s {4}'
+                      .format(self.daosctl, '0731', uid, gid, setid))
+
+        uuid_str = """{0}""".format(process.system_output(create_cmd))
+        print("uuid is {0}\n".format(uuid_str))
+        # Check if pool exists
+        self.assertTrue(
+            check_pool_files(
+                self.log, hostlist_servers, uuid_str.lower()),
+            "ERROR: Pool data not detected on servers after create")
+        self.log.info("Pool %s was successfully created", uuid_str)
+        # delete pool
         try:
-            # use the uid/gid of the user running the test, these should
-            # be perfectly valid
-            uid = os.geteuid()
-            gid = os.getegid()
-
-            # TODO make these params in the yaml
-            daosctl = self.basepath + '/install/bin/daosctl'
-
-            create_cmd = ('{0} create-pool -m {1} -u {2} -g {3} -s {4}'
-                          .format(daosctl, 0x731, uid, gid, setid))
-
-            uuid_str = """{0}""".format(process.system_output(create_cmd))
-            print ("uuid is {0}\n".format(uuid_str))
-
-            host = self.hostlist_servers[0]
-            exists = check_for_pool.check_for_pool(host, uuid_str)
-            if exists != 0:
-                self.fail("Pool {0} not found on host {1}.\n"
-                          .format(uuid_str, host))
-
             delete_cmd = ('{0} destroy-pool -i {1} -s {2}'
-                          .format(daosctl, uuid_str, setid))
+                          .format(self.daosctl, uuid_str, setid))
```

Comment:
Same as above. If possible can we use methods from daos_api.py instead of daosctl.